docker-compose/mariadb-config/prod/mariadb.cnf

```diff
@@ -0,0 +1,58 @@
+[mariadb]
+
+port                  = 3306
+bind-address          = 0.0.0.0
+
+# Fine tuning
+max_connections         = 300
```

Comment:
This is more than the default of 151. Are more than this many actively used under normal circumstances? `show global status like 'max_used_connections'` will show the maximum since startup but if its at `max_connections` then you probably had a stall or all connections waiting on a specific lock? Too many can be harmful if there isn't the CPU resources to handle this many connections.

Also if your are doing monitoring, and the monitoring user has `CONNECTION ADMIN` (to ensure it can always monitor), this might make connection to the DB hard if some reason stuck in a max_connections_used scenario. 

There's extra_port and https://mariadb.com/kb/en/thread-pool-system-status-variables/#extra_max_connections  as a potential non-default config to be ready to fix.

Reply:
Wikimedia's is a useful reference https://grafana.wikimedia.org/d/000000273/mysql?orgId=1&refresh=1m&from=now-24h&to=now

Reply:
That's super useful, but they probably use an other tool to collect metrics..